app/front/scripts/controllers/main.js

```diff
@@ -69,7 +73,24 @@ angular.module('Application')
       function initRegular() {
         $q(LoginService.tryGetToken())
           .then(function(token) {
-            return $q(osViewerService.loadDataPackages(token));
+            var packageId = null;
+            var userId = token ? LoginService.getUserId() : null;
+            if (!token) {
```

Comment:
So - logged-in user will see two controls - read-only control with currently selected package, and another one - with list with user's packages? And how user should interact with them? When user selects his own package - he will not be able to select package previously selected by URL?